File: 1013. Fibonacci Number/Solution07.java
/*
Submission Detail:{
    Difficulty : Easy
    Acceptance Rate : 69.13 %
    Runtime : 1 ms
    Memory Usage : 40.9 MB
    Testcase : 31 / 31 passed
    Ranking : 
        Your runtime beats 47.04 % of java submissions.
        Your memory usage beats 40.68 % of java submissions.
}
*/

class Solution {
    private Queue<Integer> fibQueue= new ArrayDeque<>();
    private List<Integer> fibArr = new ArrayList<>();
    
    public int fib(int n) {
        
        fibArr.add(0);
        fibArr.add(1);
        fibQueue.offer(2);
        while(!fibQueue.isEmpty()){
            int curr=fibQueue.poll();
            int fibCurr=fibArr.get(curr-1)+fibArr.get(curr-2);
            fibArr.add(fibCurr);
            if(curr<=n){
                fibQueue.offer(curr+1);
            }
        }
        return fibArr.get(n);
    }
}

File: 0344. Reverse String/Solution07.py
/*
Submission Detail:{
    Difficulty : Easy
    Acceptance Rate : 75.90 %
    Runtime : 292 ms
    Memory Usage : 22.9 MB
    Testcase : 477 / 477 passed
    Ranking : 
        Your runtime beats 34.35 % of python submissions.
        Your memory usage beats 6.06 % of python submissions.
}
*/

class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        for i, j in zip(range(0, len(s)/2), range(len(s)-1, 0, -1)):
            if(i<len(s)/2):
                temp=s[i]
                s[i]=s[j]
                s[j]=temp
           
            else:
                pass
          